cratedb_toolkit/io/iceberg.py

```
@@ -0,0 +1,121 @@
+    def load_catalog(self) -> Catalog:
+        return load_catalog(
+            self.catalog,
+            **{
+                'type': 'sql',
+                "uri": f"sqlite:///{self.path}/pyiceberg_catalog.db",
+                "warehouse": f"file://{self.path}",
+            },
+        )
+
+    @property
+    def identifier(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)